docs/style.rst

```diff
@@ -61,11 +61,11 @@ ihr in der folgenden Tabelle:
 | Modul- und Paketnamen | kurz, Kleinbuchstaben,        | ``math``, ``sys``             |
 |                       | Unterstriche nur bei Bedarf   |                               |
 +-----------------------+-------------------------------+-------------------------------+
-| Funktionsnamen        | Kleinbuchstaben, :abbr:`ggf.` | ``my_func()``                 |
+| Funktionsnamen        | Kleinbuchstaben, :abbr:`ggf.  | ``my_func()``                 |
 |                       | (gegebenenfalls)` mit         |                               |
 |                       | Unterstrichen                 |                               |
 +-----------------------+-------------------------------+-------------------------------+
-| Variablennamen        | Kleinbuchstaben, :abbr:`ggf.` | ``my_var``                    |
+| Variablennamen        | Kleinbuchstaben, :abbr:`ggf.  | ``my_var``                    |
```

Comment:
Siehe oben.

Reply:
Ok, ich habe mir jetzt mal die Datei selbst angeschaut: Die Abkürzungen passen jetzt. Lediglich in der func-Rolle solltest Du noch auf die Klammern weglassen.